pkg/utils/perm_unix.go

```diff
@@ -11,6 +11,20 @@ import (
 	"os"
 )
 
+// FileOwner is the ownership a file should have.
```

Comment:
This describes a user along with its default group, so maybe the name should be `UserGroupIDs` ?

Reply:
A file is always owned by a user and a group. To me the name is correct, and you cannot have a file owned by multiple groups at the same time.